### PR TITLE
adds Pop Shell shortcuts and notes that the other shortcuts 

### PR DESCRIPTION
**Pull request by ahoneybun**
are for releases before 20.04 LTS. This is only updating the Pop Shortcuts article.

### CODE REVIEW

File: _articles/pop-keyboard-shortcuts.md [hunk 1]
```diff
@@ -17,6 +17,20 @@ The Super Key is the OS key on your keyboard and is the Ubuntu or Pop key (<kbd>
 
 In Pop!\_OS, <kbd><span class="fl-pop-key"></span></kbd> handles the majority of OS and window actions.
 
+With the release of Pop!_OS 20.04 LTS there are new shortcuts with the introduction of Pop Shell and here are some of the shortcuts.
```
Comment (storyaddict) [inline _articles/pop-keyboard-shortcuts.md:20]: With the release of Pop!_OS 20.04 LTS and Pop Shell, there are new keyboard shortcuts available:

**Comment by WatchMkr** [comment]
While Pop!_Shell is the technology, we're calling the feature Auto Tiling. Organize shortcuts by how they're used by the customer rather than technologies. Use Pop!_Shell keyboard shortcuts for inspiration here. Then expand to include all Pop shortcuts.

(We'll be renaming Pop!_Shell shortcuts to Auto Tiling Keyboard Shortcuts.)

**Comment by ahoneybun** [comment]
> While Pop!_Shell is the technology, we're calling the feature Auto Tiling. Organize shortcuts by how they're used by the customer rather than technologies. Use Pop!_Shell keyboard shortcuts for inspiration here. Then expand to include all Pop shortcuts.
> 
> (We'll be renaming Pop!_Shell shortcuts to Auto Tiling Keyboard Shortcuts.)

That title has now been changed since they are 20.04 LTS shortcuts.

**Comment by ahoneybun** [comment]
> Should we have all the shortcuts listed in the Pop Shell Keyboard shortcuts window?

Done.

**Pull request merged**